### DISCUSSION TITLE
Edge Supports Android Extensions Now

### DISCUSSION THREAD

**Discussion by RC1992Jules** [redphx/better-xcloud#566] [General]
### Device

All devices

### Suggestion

I don't know how to report this or what this could mean for Better Xcloud on Android and it's performance, but The edge browser on Android now supports extension so I went ahead and installed tamper monkey with the latest Better Xcloud build and it's working great. I just haven't seen anybody mention anything about this and wanted to bring it up to see what that could mean for the future of Better Xcloud's development. Thank you and all of your hard work.

**Comment by KingNothing81** [comment]
I think the discussion area would be a good place for talking about this:

https://github.com/redphx/better-xcloud/discussions

Are you talking about stable Edge? Because @redphx already talked about Edge Canary supporting extensions and how to setup BxC with it on the Telegram channel.

I just wonder if mobile Edge will include some xCloud specific optimizations or if they might also make “Clarity Boost” available, now that everyone is supposed to use the web browser on mobile devices.

**Comment by RC1992Jules** [comment]
Yes, the standard or stable edge browser (not canary) that I use on my Android device is able to install tamperMonkey under the extension section and then you're able to install better x cloud just like you would on Windows. 
Also, thank you for the suggestion on where I should move this conversation.

**Comment by redphx** [comment]
#555
